README.md

```diff
@@ -1 +1,44 @@
-https://abbybudgetapp.herokuapp.com/
\ No newline at end of file
+# BUDGETLLY
+- A budgetting mobile app that allows the user to add a category and to add expenses that are associatted with the category.
+- This is a Ruby on Rails application that allows the user to:
+
+      - register and log in, so that the data is private to them.
+      - introduce new transactions associated with a category.
+      - see the money spent on each category.
+
+## Getting Started
+-Download the repository as a zip file/ clone the repository using 
+git clone https://github.com/AbbyNyakara/BudgetApp.git
+
+Create a branch and start working!
+
+## Links
+ - [Presentation Video](https://www.loom.com/share/46aa1945875b43e1938cdea5442af9e8)
```

Comment:
- [ ] Your video  should contain a highlight about a piece of the code that you find interesting or you are  proud of making.

Reply:
Thanks. I have re-done the video